Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    exception_notification (4.0.1)
+    exception_notification (4.0.2.ps.1)
```

Comment:
The precedence rules for pre-release Gem versions mean that adding a pre-release ID (after Maj.Min.Patch) makes that ordered _before_ the Maj.Min.Patch version;  i.e. 4.0.2.ps.1 < 4.0.2 (http://guides.rubygems.org/patterns/#prerelease-gems).
Remembering the 3rd ID is for a patch release, I'm working on a patch for 4.0, therefore 4.0.2.

And, background in case you were also referring to "ps1" vs. "ps.1":
Gems implement SemVer... kind-of.
SemVer would require this format to be "4.0.2-ps.1" where everything after a "-" denotes pre-release, and a dot between each identifier (see http://semver.org/ point 9 for the pre-release definition).
Precedence rules (point 11 in semver.org) however mean that you need a "." between each id that will be used to denote ordering, so if you hit "ps10" then that is ordered _before_ "ps2" because alphanum — which is wrong — but "ps.10" is ordered _after_ "ps.2".

Gems don't support a "-" before the pre-release ID's (annoyingly) but _do_ support precedence rules based on "." separators.
I admit it's a bit visually jarring, but since I'm already up to "ps.6" I think I probably did the right thing...